pkg/cmd/clustertask/testdata/Test_ClusterTask_Start_v1beta1-Dry_Run_with_--use-param-defaults_and_specified_params.golden

```diff
@@ -10,7 +10,8 @@ spec:
   - name: myarg
     value: value1
   - name: print
-    value: boom
+    value:
```

Comment:
This should remain string type. is not it ?

Reply:
The `v1beta1` should replicate what is tested in the `v1alpha1` tests. I think this was a mistake when the `v1beta1` tests were added originally.

Reply:
So to clarify, I think this should change to an array to keep it consistent.